src/components/SearchForm/SearchForm.tsx

```diff
@@ -63,6 +64,7 @@ const SearchForm = () => {
               ],
             }}
             textInputProps={{
+              isClearable: true,
```

Comment:
This is also missing a call to clear the value from the internal state. This is noticeable when you try this:
1. Search for "spaghetti"
2. Click on the clear X button
3. Click on "Search"

This returns 426 results for "spaghetti". The value is removed from the search display but is still in the state. You can pass in `isClearableCallback` and pass in a call to `setSearchTerm("")`.

Reply:
That sounds great, thanks.